lib/src/tabs-nav/Tab.tsx

```diff
@@ -103,16 +117,18 @@ const Tab = styled.a<TabStyleProps>`
   ${(props) =>
     !props.disabled &&
     `:hover {
-    background: #0000000D;
+    background: ${props.theme.unselectedBackgroundColor};
   }
 
   :focus {
-    outline: 2px solid #33aaff};
+    outline: 2px solid ${props.theme.focusOutline};
   }
 
   :active {
-    background: #0000001A;
+    background: ${props.theme.pressedBackgroundColor};
     outline: 2px solid #33aaff};
+    color: ${props.theme.selectedFontColor};
+
   }`}
```

Comment:
Remove unused `LabelContainer` styled component.

Reply:
I'm using it now no align the label and the notifications  using a flex container with row direction